test/config_test.js

```diff
@@ -76,7 +76,7 @@ suite("config", function() {
       ]
     });
 
-    assume(cfg).deep.equals(undefined);
+    assume(cfg).deep.equals({});
```

Comment:
Lodash changed behavior for  `_.defaultsDeep`, so result changed too.. But I think this result is better if all config files is missing...